### PR TITLE
Remove scheme check

### PR DESCRIPTION
**Pull request by bvennam**

### CODE REVIEW

**Comment by beemarie** [comment]
@duglin @linsun When users get the credentials for the tone analyzer service, it includes https:// as a part of the api url.  There is no reason to ask them to remove it for the env var, just to have a "use https" variable that is always set to true.

I believe this code may be a leftover from the old username/password credentials?

File: v2/analyzer-deployment.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
         - name: VCAP_SERVICES_TONE_ANALYZER_API_KEY
           value:
         - name : VCAP_SERVICES_TONE_ANALYZER_TOKEN_ADDRESS
-          value: "iam.bluemix.net/identity/token"
+          value: "https://iam.bluemix.net/identity/token"
         - name: VCAP_SERVICES_TONE_ANALYZER_SERVICE_API
           value:
         - name: USE_HTTPS
```
Comment (linsun) [inline v2/analyzer-deployment.yaml]: remove this too?

Comment (beemarie) [reply v2/analyzer-deployment.yaml]: this address is used to get the token, I just added https://

Comment (beemarie) [reply v2/analyzer-deployment.yaml]: yes, thanks!

**Comment by beemarie** [comment]
@linsun @duglin this should be ready to merge.  Even if they're approved, I don't have permissions to merge.

**Pull request merged**